examples/hybrid/staggered_nonhydrostatic_model.jl

```diff
@@ -122,6 +128,13 @@ function default_cache(Y, params)
     else
         ts_type = TD.PhaseDry{FT}
     end
+    if upwinding_mode == :first_order
+        ᶠupwind_product = ᶠupwind_product1
+    elseif upwinding_mode == :third_order
+        ᶠupwind_product = ᶠupwind_product3
+    else
+        ᶠupwind_product = nothing
+    end
```

Comment:
```suggestion
    ᶠupwind_product = if upwinding_mode == :first_order
        ᶠupwind_product1
    elseif upwinding_mode == :third_order
        ᶠupwind_product3
    else
        nothing
    end
```

Reply:
Is this just to improve legibility, or is there a niche Julia thing I'm unaware of?

Reply:
I think it's a julia thing 🙂